PiwikTracker/Tracker.swift

```diff
@@ -24,6 +24,7 @@ final public class Tracker: NSObject {
         self.queue = queue
         self.dispatcher = dispatcher
         super.init()
+        startNewSession()
         startDispatchTimer()
```

Comment:
I wouldn't call `startDispatchTimer()` directly, rather `dispatch()`. `dispatch()` would dispatch the currently stored events and starts a new timer.

In this case I don't think doing any of those is what we want. Starting a new session is not necessary coupled to dispatching events. What do you think?